prow/plugins/trigger/generic-comment_test.go

```diff
@@ -544,31 +544,6 @@ func TestHandleGenericComment(t *testing.T) {
 				},
 			},
 		},
-		{
-			name: "/retest of RunIfChanged job that doesn't need to run and hasn't run",
```

Comment:
~Shouldn't we keep this test case to ensure jobs like this are not triggered?~ This was a duplicate test case, good catch.